docs/src/browsers.md

```diff
@@ -95,6 +96,17 @@ var chromium = playwright.Chromium;
 var browser = await chromium.LaunchAsync(new BrowserTypeLaunchOptions { Channel = "chrome" });
 ```
 
+### Installing Google Chrome & Microsoft Edge
+
+If stock Google Chrome or Microsoft Edge is not available on your machine, you can install
+them using Playwright command line tool:
+
+```bash
+npx playwright install msedge
+```
+
+Run `npx playwright install --help` to see full list of the browsers that can be installed this way.
```

Comment:
**DANGER**: Google Chrome and Microsoft Edge installations will not be isolated and will be installed in the default location on your operating system.